ui/app/ot/controller/calendarViewController.js

```diff
@@ -70,6 +70,11 @@ angular.module('bahmni.ot')
                     return newVar;
                 });
                 $scope.filters.statusList = [];
+                $scope.surgeonBasedView = appService.getAppDescriptor().getConfigValue("enableSurgeonBasedView");
```

Comment:
@kavitha-sundararajan lets give this as a default option. lets not put it behind config ? its a good to have feature for product too.

Reply:
refactored the provider toggle as a default option